Cartfile.private

```diff
@@ -1,3 +1,3 @@
-github "typelift/Operadics" 
-github "typelift/SwiftCheck"
+github "typelift/Operadics" "swift-develop"
```

Comment:
You may remove these.  I've just pushed Xcode 9 updates to SwiftCheck and Operadics